package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-auth",
-  "version": "7.4.0",
+  "version": "9.1.0",
   "description": "Authentication module for central services",
   "main": "src/index.js",
   "scripts": {
```

Comment:
please add dependency check and dependency update to the package.json and make sure that all dependencies are updated